website/docs/networking/vpc-lattice/latticeabtesting.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "A/B testing with VPC Lattice"
+sidebar_position: 20
+---
+
+In this section we will show how to use Amazon VPC Lattice to perform A/B testing gradually shifting traffic from the `checkout` service to a new version `checkout-v2`.
```

Comment:
Adding some more details about the scenario would help orient the user. For example, "we have added a new shipping option as a feature and want to test it" would disclose the intent of the exercise.